tests/test_server.py

```diff
@@ -1,14 +1,18 @@
 import os
+import sys
 from pathlib import Path
 
 import pytest
+from pytest import MonkeyPatch
```

Comment:
where is this used? the package is new to me,.

Reply:
MonkeyPatch a pytest module that using for patcing mock variables and functions. I did not use it and forgot to delete it